app/uk/gov/hmrc/agentsexternalstubs/models/Services.scala

```diff
@@ -93,7 +93,14 @@ object Services {
     val json = Source
       .fromInputStream(Base64.getMimeDecoder.wrap(this.getClass.getResourceAsStream("/services.b64")), "utf-8")
       .mkString
-    Json.parse(json).as[Services].services
+    Json.parse(json).as[Services].services :+ Service(
+      "HMRC-TERSNT-ORG",
```

Comment:
We have added this enrolment already (and it should only be available to Organisations)

Reply:
cool, I will remove this my one